x/wasm/dynamic_link_test.go

```diff
@@ -477,4 +477,5 @@ func TestDynamicCallCalleeFails(t *testing.T) {
 	}
 	res, err = h(data.ctx, &executeMsg)
 	assert.ErrorContains(t, err, "Error in dynamic link")
+	assert.ErrorContains(t, err, "RuntimeError: unreachable")
```

Comment:
You added this to make sure it is panic!(), right?

Reply:
Yes. Because it is a test calling `do_panic` and it is generated `panic!()` in rust.